src/coreclr/src/jit/flowgraph.cpp

```diff
@@ -22733,7 +22733,7 @@ Compiler::fgWalkResult Compiler::fgUpdateInlineReturnExpressionPlaceHolder(GenTr
     if (retClsHnd != NO_CLASS_HANDLE)
     {
         structPassingKind howToReturnStruct;
-        var_types         returnType = comp->getReturnTypeForStruct(retClsHnd, &howToReturnStruct);
+        var_types         returnType = comp->getReturnTypeForStruct(retClsHnd, CORINFO_UNMANAGED_CALLCONV_UNKNOWN, &howToReturnStruct);
```

Comment:
I'm not sure how to determine the right calling convention here.

Reply:
We will end up here only for one managed method calling another managed method. This should be always a managed calling convention.

Reply:
Oh. Awesome! That makes it easy!